benchmark/merliner/benchmark.py

```diff
@@ -0,0 +1,48 @@
+import subprocess as sp
+
+from datetime import datetime
+
+
+def merlin_cmd(cmd, samples):
+    start_time = datetime.now()
+    merlin_file = open("bench.yaml", "w")
+
+    merlin_file.writelines(
+        'description:\n'
+        '   name: Merlin Benchmark\n'
+        '   description: Simple bench\n\n'
+    )
+
+    # Create merlin generator
+    merlin_file.writelines(
+        'merlin:\n'
+        '   samples: \n'
+        '      generate: \n'
+        '         cmd: python3 $(SPECROOT)/merliner/make_samples.py --filepath=$(MERLIN_INFO)/samples.csv --number=' + str(samples) + '\n'
+        '      file: $(MERLIN_INFO)/samples.csv\n'
+        '      column_labels: [ID]\n\n'
+    )
+
+    # Run job
+    merlin_file.writelines(
+        'study:\n'
+        '    - name: step_1\n'
+        '      description: benchmark\n'
+        '      run:\n'
+        '          cmd: %s\n' % cmd
+    )
+
+    merlin_file.close()
+    sp.call(['merlin run --local bench.yaml'], shell=True, stdout=sp.PIPE)
+    end_time = datetime.now()
+    return format(end_time - start_time)
```

Comment:
It would be better to store the measured duration as a number (e.g. a floating number of seconds) in the dataframe.